### PR TITLE
fix(tracks) Add metadata to jitsi tracks

### PR DESCRIPTION
**Pull request by robertpin**
Add timestamp to all tracks, and display surface to screen sharing tracks

### CODE REVIEW

File: modules/RTC/JitsiLocalTrack.js [hunk 1]
```diff
@@ -81,6 +81,19 @@ export default class JitsiLocalTrack extends JitsiTrack {
...
+        const displaySurface = videoType === VideoType.DESKTOP
+            ? track.getSettings().displaySurface
+            : null;
+
+        /**
+         * Track metadata.
+         */
+        this.metadata = {
+            timestamp: Date.now(),
+            ...displaySurface ? { displaySurface } : {}
```
Comment (saghul) [inline modules/RTC/JitsiLocalTrack.js:93]: Parentheses for extra readability?

**Review by saghul — APPROVED** [review]
One nit, LGTM otherwise.

File: modules/RTC/JitsiLocalTrack.js (hunk 1 above)
Comment (robertpin) [reply modules/RTC/JitsiLocalTrack.js:93]: Tried. Eslint doesn't like them, says they're unnecessary

Comment (saghul) [reply modules/RTC/JitsiLocalTrack.js:93]: ah right

**Pull request merged**